package.json

```diff
@@ -78,7 +78,8 @@
     "react-test-renderer": "^18.2.0",
     "style-loader": "^3.3.1",
     "ts-jest": "^29.0.3",
-    "webpack": "^5.65.0"
+    "webpack": "^5.65.0",
```

Comment:
Why `webpack` is required here?

Reply:
I have referred below it seems like the current version of next JS uses 
https://nextjs.org/docs/upgrading#webpack-5
[why webpack as devDependency](https://dev.to/thawkin3/understanding-dev-peer-and-regular-dependencies-gmi#:~:text=For%20example%2C%20webpack%20and%20rollup,karma%20%2C%20mocha%20%2C%20or%20jasmine%20.), 
Anyways i haven't added it ,the change is shown i think because of comma, if you think this needs to be investigated i can create a ticket

Reply:
Which change is this?